src/index.js

```diff
@@ -42,7 +42,7 @@ import Item from 'src/views/Items/Item';
 import Items from 'src/views/Items/Items';
 import Statistic from 'src/views/Statistic/Statistic';
 
-export default {
+const stardust = {
```

Comment:
This style export was not valid ES module syntax.  Babel has dropped support for it (common js interop).  Though we technically have the plugin to support it, we only have it because of karma's need to use the babel register hook.  Updated to proper ES module syntax.